include/translatable-object.php

```diff
@@ -422,12 +429,14 @@ public function sanitize_int_ids_list( $ids ) {
 	 *
 	 * @param int[] $language_ids List of language `term_taxonomy_id`.
 	 * @param int   $limit        Max number of objects to return. `-1` to return all of them.
+	 * @param array $args         The object args.
 	 * @return string
 	 *
 	 * @phpstan-param array<positive-int> $language_ids
 	 * @phpstan-param -1|positive-int $limit
+	 * @phpstan-param array<empty> $args
```

Comment:
Good catch, it will help us to spot code that adds arguments that shouldn't be there 👍